src/Sentry/Internal/Http/RetryAfterHandler.cs

```diff
@@ -85,7 +85,7 @@ private DateTimeOffset GetRetryAfterTimestamp(HttpResponseMessage response)
         // Sentry was sending floating point numbers which are not handled by RetryConditionHeaderValue
         // To be compatible with older versions of sentry on premise: https://github.com/getsentry/sentry/issues/7919
         if (response.Headers.TryGetValues("Retry-After", out var values)
-            && double.TryParse(values.FirstOrDefault(), out var retryAfterSeconds))
+            && double.TryParse(values.FirstOrDefault(), NumberStyles.Float | NumberStyles.AllowThousands, CultureInfo.InvariantCulture, out var retryAfterSeconds))
```

Comment:
The `NumberStyles` parameter is mandatory when providing an `IFormatProvider`, so I picked the default value for ``NumberStyles` for the current overload of `TryParse`.